### PR TITLE
Fix Nginx example config

### PR DESCRIPTION
**Pull request by shaftoe**
check https://stackoverflow.com/questions/15423500/nginx-showing-blank-php-pages#16887296 for the reason why

### CODE REVIEW

File: mods/sample-nginx.config [hunk 1]
```diff
@@ -88,6 +88,11 @@ server {
     rewrite ^/(.*) /index.php?pagename=$uri last;
   }
 
+  # add rewrite needed by install process
+  if (!-f $request_filename) {
+      rewrite ^/fd/(.+)$ /fd/index.php?q=$1 last;
```
Comment (annando) [inline mods/sample-nginx.config]: What is this ```/fd``` folder?

File: mods/sample-nginx.config [hunk 2]
```diff
@@ -129,9 +134,8 @@ server {
     # With php5-fpm:
     fastcgi_pass unix:/var/run/php5-fpm.sock;
 
-    include fastcgi_params;
+    include fastcgi.conf;
```
Comment (annando) [inline mods/sample-nginx.config]: Where is this config file?

**Comment by annando** [comment]
This ```/fd``` thing doesn't make sense to me. Can you explain it?

**Comment by annando** [comment]
Sorry, had accidentally closed it.

File: mods/sample-nginx.config (hunk 1 above)
Comment (shaftoe) [reply mods/sample-nginx.config]: Cargo-culted from https://github.com/friendica/friendica/issues/868

**Comment by shaftoe** [comment]
Check https://github.com/friendica/friendica/issues/868 thread for why of the redirect, without that the installer was complaining of missing redirect features from `.htaccess`

**Comment by annando** [comment]
I hope that someone with more experience in nginx can review this because this looks strange to me. :-)

**Comment by shaftoe** [comment]
Yeah I get it, actually that fix doesn't work on a new host so might not be right. The fastcgi part though stands still

**Comment by shaftoe** [comment]
If I understand the code, the installer tries to GET "{{ base_url }}/install/testrewrite" and if it doesn't return "ok" string the check fails, so yeah, the `/fd` part looks completely bogus :)

**Pull request merged**